### PR TITLE
Windows dlldeps

### PR DESCRIPTION
**Pull request by tgingold**
Add makefile recipes to add dll dependencies in the install directory.
Could be used to build standalone windows package.

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -225,6 +225,11 @@ uninstall.mcode.program:
 
 install.mcode: install.mcode.program install.vhdllib
 
+install.mcode.dll_deps:
```
Comment (Paebbels) [inline Makefile.in:228]: Don't you need to call this target somewhere?

Dependencies of DLLs can be recursive. See here:  
image

I used https://github.com/lucasg/Dependencies to find dependencies on Windows.

**Comment by tgingold** [comment]
So the target has to be called when we want to build a standalone package.
It will be called manually.

Dependencies can be recursive, but in practice they aren't: all libgnat dependencies are also in the libghdl dll.

**Comment by tgingold** [comment]
More precisely I suppose that ldd handle does the recursion.

**Comment by Paebbels** [comment]
Hmmm when testing `ldd` I see a recursive resolution of DLLs. Strange ... why have I spend time on writing a recursive Bash script ... Or did I follow some online article? ... strange strange strange

-------
Can we bundle these new targets with `--enable-standalone` in configure, so this target can be activated? I have added this in my other PR and it works like a charm.

**Pull request merged**